demos/segmentation_demo/cpp/main.cpp

```diff
@@ -188,6 +194,33 @@ cv::Mat renderSegmentationData(const ImageResult& result, OutputTransform& outpu
     return output;
 }
 
+void print_raw_results(const ImageResult& result, std::vector<std::string> labels)
+{
+
+    slog::info << " Class ID | Pixels | Percentage " << slog::endl;
+    cv::Mat histogram;
+    double min_val, max_val;
+    cv::minMaxLoc(result.resultImage, &min_val, &max_val);
+    int max_classes = static_cast<int>(max_val) + 1; // We use +1 for only background case
+    const float range[] = { 0, max_classes };
+    const float * ranges = { range };
+    cv::calcHist(&result.resultImage, 1, 0, cv::Mat(), histogram, 1, &max_classes, &ranges);
+    const double all = result.resultImage.cols * result.resultImage.rows;
+    for (int i = 0; i < max_classes; ++i)
+    {
+        const int value = histogram.at<int>(i);
+        if (value > 0)
+        {
+            std::string label = i < labels.size() ? labels[i] : "#" + std::to_string(i);
+            slog::info << " "
+                << std::setw(7) << label << " | "
+                << std::setw(6) << histogram.at<float>(i) << " | "
+                << std::setw(6) << histogram.at<float>(i) / all * 100 << "%"
```

Comment:
You should fix precision similar to python

Reply:
Done

Reply:
They are still different. It should be
`<< std::setw(5) << std::setprecision(2) << std::fixed << std::right << value / all * 100 << "%"`

Reply:
Thanks, done!